qiskit_experiments/calibration_management/backend_calibrations.py

```diff
@@ -196,7 +213,149 @@ def export_backend(self) -> Backend:
 
         backend.defaults().qubit_freq_est = self.get_qubit_frequencies()
         backend.defaults().meas_freq_est = self.get_meas_frequencies()
-
-        # TODO: build the instruction schedule map using the stored calibrations
+        backend.default().instruction_schedule_map = self._inst_map
 
         return backend
+
+    def inst_map_add(
```

Comment:
I think we can also move this logic to instmap with https://github.com/Qiskit/qiskit-terra/pull/6954 (currently it is waiting a review). Basically, we can set all schedule with parameter + default value, and call it with parameter assignment. Above PR allows to write

```python
instmap.add("sx", (0, ), sx_sched, arguments={"duration": 160, "amp": 0.1, "sigma": 60})  # in cal
instmap.get("sx", (0, ), amp=Parameter("amp"))  # from experiment
```

but we need separate logic that maps gate names, i.e. amp-parametrized "sx" = "rabi".

Reply:
Sounds good to me. We can always adapt depending on which PR goes in first. Once PR 6954 is in we will probably not need the function `inst_map_add`. We could do:
```
cals.instruction_schedule_map.add("Rabi", (0, ), cals.get_schedule("sx", 0))
```
Here`sx` can probably be an experiment option. One question: the schedule above does not have a parameter in it. How would the transpiler pass with the inst_map work? I.e. currently I can do:
```python
inst_map = InstructionScheduleMap()

param = Parameter("amp")

with pulse.build() as xp:
    pulse.play(pulse.GaussianSquare(240, param, 60, 60), pulse.DriveChannel(0))
    
inst_map.add("rx", (0, ), xp, arguments=["amp"])

qc = QuantumCircuit(1)
qc.rx(0.1234, 0)

qc2 = transpile(qc, inst_map=inst_map)
print(qc2.calibrations)
```
Would the above still work with PR 6954 if the schedule does not explicitly have a parameter? I.e. if the schedule was
```python
with pulse.build() as xp:
    pulse.play(pulse.GaussianSquare(240, 0.5, 60, 60), pulse.DriveChannel(0))
```

Reply:
No that doesn't work. In the instmap, to create parameter signature object, we need some unique name associated with the pulse parameter. In the last example, `0.5` doesn't have a name and we cannot create signature object from it. We can implicitly infer the name from signature of `GaussianSquare`, i.e. 0.5 should be `amp`, however, if we have multiple pulses in a schedule, there will be name conflict issue. 

Recently I lean toward introducing pulse library in instmap, so that we can clearly separate pulse and schedule.

Reply:
Ok. Based on this discussion we leave `inst_map_add` in `BackendCalibrations` for now.